controllers/grid/users/stageParticipant/StageParticipantGridHandler.inc.php

```diff
@@ -114,6 +114,32 @@ function initialize($request) {
 			null,
 			$cellProvider
 		));
+		$submission = $this->getSubmission();
+		$submissionId = $submission->getId();
+		if ( Validation::isLoggedInAs() ) 
+		{
```

Comment:
We put the `{` on the same line as the conditional statement and don't add spaces inside the parentheses:

```
if (Validation:isLoggedInAs()) {
```